pollution related issues

```diff
@@ -0,0 +1,114 @@
+<html>
```

Comment:
Overall, the HTML code is well structured, but there are a few points that could be improved for better code readability and maintainability:

1. **Empty elements**: There are empty anchor tags and list items in the navigation section. If these are not needed, they should be removed.

```html
<ul class="pull-left">
  <a></a></li>
</ul>
<ul class="pull-right">
  <li><a href="#"></a></li>
</ul>
```

2. **Missing tags**: The opening tag for the list item `<li>` is missing in the navigation section.

```html
<ul class="pull-left">
  <a></a></li> <!-- Missing <li> tag -->
</ul>
```

3. **Long paragraphs**: The paragraphs in the "Info" section are quite long. Breaking them into smaller paragraphs would make them easier to read.

4. **Code indentation**: The HTML code is not consistently indented, which makes it harder to read. It's important to keep a consistent indentation level for readability and maintainability.

5. **Alt attribute for images**: The `<img>` tags are missing the `alt` attribute. This attribute is important for accessibility and SEO. It provides a text description of the image if it cannot be displayed.

```html
<img src="http://tunza.eco-generation.org/file/image8.jpg">
<!-- Should be -->
<img src="http://tunza.eco-generation.org/file/image8.jpg" alt="description of image">
```

6. **Validation**: You should validate your HTML to ensure it follows the HTML standard and doesn't have any syntax errors. You can use an online tool like the W3C Markup Validation Service.

7. **Embedded Video**: The YouTube video iframe could have a title attribute for better accessibility.

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/_R0vxCmls6o" frameborder="0" allowfullscreen></iframe>
<!-- Should be -->
<iframe title="video title" width="560" height="315" src="https://www.youtube.com/embed/_R0vxCmls6o" frameborder="0" allowfullscreen></iframe>
``